pom.xml

```diff
@@ -225,6 +210,11 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.jetbrains</groupId>
+			<artifactId>annotations</artifactId>
+			<version>24.1.0</version>
+		</dependency>
```

Comment:
this may be a typo. Sometimes happens with the NonNull annotations, check it out please.